src/components/PeopleList.tsx

```diff
@@ -1,57 +1,30 @@
-import axios, { AxiosPromise, AxiosRequestConfig, AxiosResponse } from 'axios';
 import * as React from 'react';
 import { Letter } from './Letter';
+import { IEmployee } from './MainComponent';
 
-export interface IEmployee {
-	_id: string;
-	email: string;
-	name: string;
-	phone: string;
-}
-
-interface IPeopleListState {
+interface IPeopleListProps {
 	employees: IEmployee[];
+	letters: string[];
 }
 
-export class PeopleList extends React.Component<{}, IPeopleListState> {
-	private letters: string[];
+export class PeopleList extends React.Component<IPeopleListProps, {}> {
+	public state: IPeopleListProps;
 
-	constructor(props: object) {
+	constructor(props: IPeopleListProps) {
 		super(props);
-		this.state = { employees: [] };
-		axios.get(`http://localhost:3000/api/database/employees`).then((res: AxiosResponse) => {
-			const users: IEmployee[] = res.data;
-			this.setState({ employees: users });
-		});
+		this.state = {
+			employees: this.props.employees,
+			letters: this.props.letters,
+		};
+	}
 
-		this.letters = [
-			'A',
-			'B',
-			'C',
-			'D',
-			'E',
-			'F',
-			'G',
-			'H',
-			'I',
-			'J',
-			'K',
-			'L',
-			'M',
-			'N',
-			'O',
-			'P',
-			'Q',
-			'R',
-			'S',
-			'T',
-			'U',
-			'V',
-			'W',
-			'X',
-			'Y',
-			'Z',
-		];
+	public componentWillReceiveProps(nextProps: IPeopleListProps): void {
```

Comment:
Having a test for this could be useful to make sure someone who doesn't understand why it's here doesn't accidentally remove it.